tests/service/ParserTests.fs

```diff
@@ -368,3 +369,65 @@ match () with
     match exprs with
     | SynExpr.Match(_, _, [_; _], _, _) -> ()
     | _ -> failwith "Unexpected tree"
+
+
+[<Test>]
+let ``Dot lambda - Recovery _.`` () =
```

Comment:
Is there any reason not to use `SyntaxTree` tests and do the manual checks on the tree? 🙂